tinygrad/ops.py

```diff
@@ -269,7 +269,9 @@ def _toposort(u:UOp, cache:dict[UOp, dict[UOp, None]]):
     return _toposort(self, cache={})
 
   @functools.cached_property
-  def tuplize(self:UOp) -> tuple[int, Any, Optional[DType], tuple]: return (self.op.value, self.arg, self.dtype, tuple(x.tuplize for x in self.src))
+  def tuplize(self:UOp) -> tuple:
+    if self.op in GroupOp.Binary and self.src[1].op in (Ops.CONST, Ops.VCONST): return self.src[0].tuplize + (self.op.value, self.src[1].arg)
```

Comment:
Why do we need a special case here?

Reply:
This is part of #8123, by itself it doesn't achieve that much but one you start ordering a chain you want x and x*const to be sorted next to each other so that it can get folded to x*(const+1)

Reply:
Also should sort x//c*c and x%c very close to each other